tests/php/modules/shortcodes/test-class.facebook.php

```diff
@@ -24,4 +32,90 @@ public function test_shortcodes_facebook() {
 		$this->assertNotEquals( $content, $shortcode_content );
 	}
 
+	/**
+	 * Test a Facebook Video using the old, first format (video.php)
+	 *
+	 * @covers ::jetpack_facebook_embed_handler
+	 *
+	 * @since 7.5.0
+	 */
+	public function test_shortcodes_facebook_video_old() {
+		global $post;
+
+		$fb_video_id = '546877659119730';
+		$url         = 'https://www.facebook.com/video.php?v=' . $fb_video_id;
+		$post        = $this->factory()->post->create_and_get( array( 'post_content' => $url ) );
+
+		setup_postdata( $post );
+
+		// Test HTML version.
+		ob_start();
+		the_content();
+		$actual = ob_get_clean();
+		$this->assertContains(
+			sprintf(
+				'<div class="fb-video" data-allowfullscreen="true" data-href="%s"></div>',
+				$url
+			),
+			$actual
+		);
+	}
+
+	/**
+	 * Test a Facebook Video using the new format (watch/)
+	 *
+	 * @covers ::jetpack_facebook_embed_handler
+	 *
+	 * @since 7.5.0
+	 */
+	public function test_shortcodes_facebook_video_new() {
+		global $post;
+
+		$fb_video_id = '546877659119730';
+		$url         = 'https://www.facebook.com/watch/?v=' . $fb_video_id;
```

Comment:
would this pass without the trailing `watch/` slash?

Reply:
Now it will, as of f6423c326f81c8e4c558f0d4cb3f5d4c9b0a354d. I don't think this deserves its own test though. What do you think?